src/app/services/password/password-reset.service.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { TestBed } from '@angular/core/testing';
+import {HttpTestingController, HttpClientTestingModule } from '@angular/common/http/testing';
+import { PasswordResetService } from './password-reset.service';
+import { APPCONFIG } from 'src/app/config';
+
+describe('PasswordResetService', () => {
+  let httpMock: HttpTestingController;
+  const url = `${APPCONFIG.base_url}/auth/password-reset/`
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+    imports: [HttpClientTestingModule ],
+  });
+    httpMock = TestBed.get(HttpTestingController);
+  });
+
+  it('should be created', () => {
+    const service: PasswordResetService = TestBed.get(PasswordResetService);
+    expect(service).toBeTruthy();
+  });
+
+  it('should trigger a service passing', () => {
+    const service: PasswordResetService = TestBed.get(PasswordResetService);
+    const mockData = {
+      email: 'joel@andela.com'
+    };
+    service.getResetLink(mockData).subscribe();
+    const req = httpMock.expectOne(url);
+    req.flush(mockData);
+    });
+
+  it('should trigger a service failing', () => {
```

Comment:
Hello @gye, This description does not appropriately explain the test, You could talk about how the service should still be called when an email is invalid.